eng/pipeline/stages/run-stage.yml

```diff
@@ -193,6 +193,11 @@ stages:
                   $env:PATH = (
                     $env:PATH -split ';' | Where-Object { $_ -ne 'C:\Program Files\Git\usr\bin' }
                   ) -join ';'
+
+                  Write-Host "Removing Chocolatey shim for SWIG to avoid running misc SWIG tests that would fail."
+                  # The Chocolatey shims are located in a single folder in PATH, so we can't change PATH to exclude it.
+                  # Upstream Windows builders don't have SWIG installed, so this makes coverage even.
+                  Remove-Item (Get-Comment swig).Source
```

Comment:
I mentioned in the Go sync that I thought we could use the PATH reassignment trick, but it turns out that the SWIG shim the build finds/uses is installed into the shared Chocolatey bin folder with all the other shims. This is where find gcc, etc. so changing PATH doesn't work. Deleting the shim should be fine--our builds are on one-time-use VMs.

I also considered `choco uninstall`, but that takes a while.